redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/EventListener.kt

```diff
@@ -84,48 +84,61 @@ public open class EventListener {
   }
 
   /**
-   * Invoked on a request to create an unknown widget [kind].
+   * Invoked on a request to create an unknown widget with [tag]. This is a schema mismatch and the
+   * widget is ignored.
```

Comment:
@JakeWharton is doing nothing here safe?

Reply:
Hmm I didn't write these so I'm not quite sure how this gets invoked. From what I would guess, the decision was already made at the Redwood layer to return `null` (i.e., ignore the widget) and here you can either log or escalate that to a throw.